### DISCUSSION TITLE
Update or upgrade my running sbfspot on a rhaspberry pi

### DISCUSSION THREAD

**Discussion by Asesterhenn** [SBFspot/sbfspot-config#64] [General]
Hi there,

I would like to update or upgrade my running sbfspot on a rhaspberry pi

I am not sure whic version is currently installed (where can I check for that ... I know dummy question:-))

From the wiki I read to run the sbf-config.
I started it but ran immediately into problems because the script could not find any of my 3 (!) cfg files (SMA1.cfg ... SMA3.cfg) and asked me to start a new config.

What can I do to get any further without fully reinstalling / reconfiguring my setup?

Thank you

Andreas

**Comment by sbf-** [comment]
> I am not sure whic version is currently installed (where can I check for that?)

`SBFspot -?` shows some help and also the version





> the script could not find any of my 3 (!) cfg files (SMA1.cfg ... SMA3.cfg)

sbfspot-config searches for SBFspot.cfg (default config file)

Best option is to update manually: download the tar, extract the files and replace SBFspot and SBFspotUpload executables
